### PR TITLE
statement: Save columns options

### LINKED ISSUE #410: Remember Column Options
## Feature Request

**Is your feature request related to a problem? Please describe:**

image

We can remember column options in local storage so that user don't need to repeatedly set it.

### PR DESCRIPTION
**Pull request by baurine**
Fix #410 , save statements column options to local storage.

### CODE REVIEW

File: ui/lib/apps/Statement/components/StatementsOverview.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {
   Menu,
   Checkbox,
 } from 'antd'
+import { CheckboxChangeEvent } from 'antd/lib/checkbox'
```
Comment (breeswish) [inline ui/lib/apps/Statement/components/StatementsOverview.tsx:12]: prefer to use `antd/es/checkbox`

File: ui/lib/apps/Statement/components/StatementsOverview.tsx [hunk 2]
```diff
@@ -196,16 +201,20 @@ export default function StatementsOverview({
...
-  }>({
-    digest_text: true,
-    sum_latency: true,
-    avg_latency: true,
-    exec_count: true,
-    avg_mem: true,
-    related_schemas: true,
-  })
+  }>(() =>
+    load(VISIBLE_COLUMN_KEYS, {
```
Comment (breeswish) [inline ui/lib/apps/Statement/components/StatementsOverview.tsx:205]: you can use https://hooks.umijs.org/state/use-local-storage-state

Comment (baurine) [reply ui/lib/apps/Statement/components/StatementsOverview.tsx:205]: cool! will try it soon!

Comment (baurine) [reply ui/lib/apps/Statement/components/StatementsOverview.tsx]: nice hook!

**Pull request merged**